frang/test_http_resp_code_block.py

```diff
@@ -0,0 +1,62 @@
+"""Functional tests for http_resp_code_block."""
```

Comment:
`http_resp_code_block` is defined in wiki, but there should be a good test description, what is happening and what behaviour is expected.